### PR TITLE
Updates for Azure Batch Data Plane SDK 7.0

### PR DESCRIPTION
**Pull request by matthchr**
## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [ ] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [ ] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### Note for reviewers
**Link to Swagger this is based on:** https://github.com/Azure/azure-rest-api-specs/pull/1205
**This change has been previously reviewed [here](https://github.com/Azure/azure-sdk-for-net-pr/pull/728) by @markcowl** - note that the commit hashes for all of the commits except the most recent are the same as in the link.
**Please review ASAP as this is for a //build release**

### CODE REVIEW

**Comment by matthchr** [comment]
@markcowl, @shahabhijeet  - Can you take a quick look at this and merge or point me at who should look at it? I want to publish by 5/10.

Please note, this was already reviewed (see "Note for reviewers" section above).

File: src/SDKs/Batch/DataPlane/Azure.Batch/Azure.Batch.csproj [hunk 1]
```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <PackageId>Azure.Batch</PackageId>
     <Description>This client library provides access to the Microsoft Azure Batch service.</Description>
-    <VersionPrefix>6.1.0</VersionPrefix>
+    <VersionPrefix>7.0.0</VersionPrefix>
```
Comment (shahabhijeet) [inline src/SDKs/Batch/DataPlane/Azure.Batch/Azure.Batch.csproj]: @matthchr can you remove any traces of PORTABLE compiler constant.
Also remove any commented code.

**Review by shahabhijeet — CHANGES_REQUESTED** [review]
This is good to merge once you address few minor edit that are required.

**Comment by matthchr** [comment]
@shahabhijeet - Thanks for taking a look. I have updated the csproj to clean it up a bit. I left one section of commented out code because once your fix comes in for allowing builds of non net452/netstandard1.4 projects I will re-enable that commented out bit.

As for `portable`, in order to remove it I must update the version of AutoRest we are using. I didn't do that with this release because of time constraints around meeting the //build deadline, but it is on my list.

Can we merge this PR now, and I will tackle fixing the `portable` issue later? We need to meet //build deadlines so I had this PR previously signed off the @markcowl already (see "Note for reviewers in original PR message").

**Pull request merged**